.editorconfig

```diff
@@ -0,0 +1,4 @@
+[*.{kt,kts}]
+ktlint_disabled_rules = no-wildcard-imports
+ij_kotlin_allow_trailing_comma_on_call_site = false
+ij_kotlin_allow_trailing_comma = false
```

Comment:
nitpick: I would prefer the exact opposite rules 🙈 but feel free to leave it be for now

Reply:
But this is what we have in the other projects I know of (ocpp, change-log cli)